backend/experiments/LibExperimentalStdLib/LibCLI.fs

```diff
@@ -75,6 +75,51 @@ let fns : List<BuiltInFn> =
       previewable = Impure
       deprecated = NotDeprecated }
 
+    { name = fn "System" "getEnv" 0
```

Comment:
How about `Environment.get`?

Reply:
I feel like `Environment.get` should contain the word "variable" or "var" somewhere. Otherwise, it seems too general. Maybe `Environment.getVar` or something.

I'll look tomorrow at how other standard libraries name such functions

Reply:
Good point. It's not really Environment, it's really Environment Variables. Maybe "EnvVar" or "getVar" might work, yeah.

Reply:
Used `EnvVar.get` and `EnvVar.getAll`